.travis/check-generate.sh

```diff
@@ -5,6 +5,7 @@ go generate gonum.org/v1/gonum/blas/gonum
 go generate gonum.org/v1/gonum/unit
 go generate gonum.org/v1/gonum/graph/formats/dot
 if [ -n "$(git diff)" ]; then
+	git checkout -- go.mod # Discard changes to go.mod that have been made.
```

Comment:
we possibly might have also to discard `go.sum` ?

Reply:
When we commit it, yes. It's not currently in the repo.